ts/src/client.ts

```diff
@@ -101,7 +101,14 @@ class GrpcClient<TRequest extends ProtobufMessage, TResponse extends ProtobufMes
       const gRPCMessage = headers.get("grpc-message") || [];
       this.props.debug && debug("onHeaders.gRPCMessage", gRPCMessage);
       if (code !== Code.OK) {
-        this.rawOnError(code, gRPCMessage[0]);
+        let statusMessage;
+        try {
+          statusMessage = this.decodeGRPCStatus(gRPCMessage[0]);
+        } catch (err) {
+          this.rawOnError(Code.Internal, "Cannot decode gRPC-Message header");
```

Comment:
Is this the correct error code? Should it be unknown as it is an error on the transport library rather than the end users code?

Reply:
It's unspecified; maybe it should be Unknown but anyway that only happens when a corrupted/broken message is received. Any reference?

Reply:
Only from the discussion in https://github.com/improbable-eng/grpc-web/pull/150

Reply:
Aligned the behavior with grpc-go.